conf/views/viewAllArrivals.njk

```diff
@@ -20,6 +21,10 @@
         </a>
       </p>
 
+      {%- if results.totalPages != 1 %}
```

Comment:
I'm not sure I understand this guard here, why wouldn't we show this if there is totals pages is 1? Wouldn't we still need to show the information "Showing page 1 to 99"? We may want to run this past Design

Reply:
Good point Amaal, we have had a chat with design they like this idea. Will implement.